electron-app/start-backend.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
```

Comment:
This is super simple. Why use node when you are just starting a python process. Just write it in python. I can help uf you need

Reply:
Well, it is part of the front-end tooling, and I am more used to write js/ts than python :D

If you think it makes sense to re-write it in python I am also ok with that. Basically the only requirement is to be able to handle both POSIX and WIN32 paths and environments (get the systems tmp folder) so I don't have any strong feelings.

Reply:
Yeah please write it in python. Frontend tooling or not it's just a bit silly write a node script to call a python process.

It will do you good to practice this in python :heart: -- I am here to help with writing it if you need to ask qs (either discord or whatsapp)

Reply:
Hmm no forget it. I now understand that this script needs to call python itself so makes no difference in what it's written in.